app/services/channel_queues/response_retriever.rb

```diff
@@ -32,7 +31,7 @@ def self.retrieve(original_request_text, channel_id, channel_name, user_id, user
         ChannelQueueMembership.create!(user: user, channel_queue: channel_queue)
         json_result[:text] = "#{user.slack_user_name} joined queue for #{channel_queue.slack_channel_name}."
         json_result[:response_type] = 'in_channel'
-      when 'leave', 'charging'
```

Comment:
Should we keep the leave command for people who want to drop out of the queue but haven't actually gone to charge?

Reply:
yeah i agree, i just think it might be good to rename it since David pointed out the confusion between `leave` and `moved`

Reply:
put it back (dropped the commit to remove it) https://github.com/choruk/foos_portal/compare/5ba1b15bef3e5d620e34114a1fd2f9490b9d0f85..3fa64fe4ec62c51444f1044fca5cb017d4421517